dags/search_rollup.py

```diff
@@ -30,11 +29,17 @@
 
 
 def add_search_rollup(dag, mode, instance_count, upstream=None):
-    """Create a search rollup for a particular date date"""
+    """Create a search rollup for a particular date date
+
+    This can be called with an optional DAG passed into `upstream`. The rollup
+    job will inherit the default values of the referenced DAG.
+    """
 
     search_rollup = EMRSparkOperator(
         task_id="search_rollup_{}".format(mode),
         job_name="{} search rollup".format(mode).title(),
+        owner="amiyaguchi@mozilla.com",
+        email=default_args["email"],
```

Comment:
So, wait, default_args doesn't apply the email default arg? 😧

Reply:
Oh wait, I see.. you're referencing the `default_args` variable in the file. Hmm, I think in an ideal world we'd pass in those parameters to the `add_search_rollup` function, or pass in `default_args` with the double-star